pkg/image/admission/admission.go

```diff
@@ -85,7 +84,11 @@ func (a *imageLimitRangerPlugin) Admit(attr admission.Attributes) error {
 		return nil // not applicable
 	}
 
-	return a.limitRanger.Admit(attr)
+	err := a.limitRanger.(admission.MutationInterface).Admit(attr)
+	if err != nil {
+		return err
+	}
+	return a.limitRanger.(admission.ValidationInterface).Validate(attr)
```

Comment:
why not add another Validate to imageLimitRangerPlugin? At least a TODO would make sense here.

Reply:
> why not add another Validate to imageLimitRangerPlugin? At least a TODO would make sense here.

I'm trying to avoid messing with the chain for the rebase. We'll refactor them all eventually.